pkg/lib/regex/regex.go

```diff
@@ -40,3 +40,28 @@ var _alphaNumericDashUnderscoreRegex = regexp.MustCompile(`^[a-zA-Z0-9_\-]+$`)
 func IsAlphaNumericDashUnderscore(s string) bool {
 	return _alphaNumericDashUnderscoreRegex.MatchString(s)
 }
+
+// used the evaluated form of
+// https://github.com/docker/distribution/blob/3150937b9f2b1b5b096b2634d0e7c44d4a0f89fb/reference/regexp.go#L68-L70
+
```

Comment:
nit: this new line is not necessary since the comment refers to the variable below

Reply:
Done!